src/mixins/nft.js

```diff
@@ -184,6 +184,9 @@ export default {
         this.NFTClassMetadata.account_owner
       );
     },
+    iscnAuthor() {
```

Comment:
this name might cause potential confusion in the future but i have no good suggestion...
`iscnMetadataAuthor` ?

Reply:
How about iscnWorkAuthor that imply Author under CreativeWork in schema.org

Reply:
makes sense

Reply:
Updated